sdk/core/azure-core/src/test/java/com/azure/core/handler/JUL2SLF4JHandlerTest.java

```diff
@@ -0,0 +1,242 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.handler;
+
+import com.azure.core.util.Configuration;
+import com.azure.core.util.logging.LogLevel;
+import org.junit.jupiter.api.Test;
+import java.util.Arrays;
+import java.util.Optional;
+import java.util.logging.LogManager;
+import java.util.logging.Logger;
+import java.util.logging.LogRecord;
+import java.util.logging.Level;
+import java.util.logging.Handler;
+import static com.azure.core.util.Configuration.PROPERTY_AZURE_LOG_LEVEL;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * Unit tests for {@link JUL2SLF4JHandler}.
+ */
+public class JUL2SLF4JHandlerTest {
+
+    private static final Configuration CONFIGURATION = Configuration.getGlobalConfiguration();
+    private static Boolean isRootLogger = true;
+
+    @Test
+    public void jul2slf4jTest() {
+        jul2slf4jTest4LogManager();
+        jul2slf4jTest4Logger();
+    }
+
+    private void jul2slf4jTest4LogManager() {
```

Comment:
These should be broken up into smaller test cases because we want to be able to understand if something fails in between.

You can use `@BeforeEach` and `@AfterEach` or `@BeforeAll`  and `@AfterAll` if you want a global or set-up to happen before each test.

Reply:
According to your comment, Test cases are split in the latest version .